packages/manager/apps/public-cloud/src/index.routes.js

```diff
@@ -5,56 +5,62 @@ import { DEFAULT_PROJECT_KEY } from './index.constants';
 export default /* @ngInject */ ($stateProvider, $urlRouterProvider) => {
   $stateProvider.state('app', {
     url: '/?onboarding',
-    redirectTo: (trans) => {
-      const $q = trans.injector().get('$q');
-      const publicCloud = trans.injector().get('publicCloud');
-      return $q
-        .all([
-          publicCloud.getDefaultProject(),
-          publicCloud.getServices([
-            {
-              field: 'route.path',
-              comparator: 'eq',
-              reference: '/cloud/project/{serviceName}',
-            },
-            {
-              field: 'billing.lifecycle.current.state',
-              comparator: 'eq',
-              reference: 'unpaid',
-            },
-          ]),
-          publicCloud.getProjects([
-            {
-              field: 'status',
-              comparator: 'like',
-              reference: 'suspended',
-            },
-          ]),
-        ])
-        .then(([defaultProjectId, unPaidProjects, suspendedProjects]) => {
-          if (!isEmpty(suspendedProjects) || !isEmpty(unPaidProjects)) {
-            return { state: 'pci.projects' };
-          }
-          if (defaultProjectId) {
-            return {
-              state: 'pci.projects.project',
-              params: {
-                projectId: defaultProjectId,
-              },
-            };
-          }
-          return {
-            state: trans.params().onboarding
-              ? 'pci.projects.onboarding'
-              : 'pci.projects.new',
-          };
-        });
-    },
+    redirectTo: 'app.redirect',
     resolve: {
       rootState: () => 'app',
     },
   });
 
+  $stateProvider.state('app.redirect', {
```

Comment:
Maybe we could add a comment following this PR description so we avoid removing this in some near future thinking it is non sense

Reply:
@cbourgois do you plan on documenting this change ?

Reply:
Done in a dedicated commit. 
Could be squashed.